index.d.ts

```diff
@@ -8,6 +8,8 @@ export const findUpStop: unique symbol;
 
 export type Match = string | typeof findUpStop | undefined;
 
+export type Options = LocatePathOptions & {stopAt?: string};
```

Comment:
The `stopAt` option needs docs: https://github.com/sindresorhus/typescript-definition-style-guide

Reply:
Added docs:
https://github.com/sindresorhus/find-up/pull/54/commits/c2134ed942e32804dd16c07a28e40d5e4809c521